franka_example_controllers/src/cartesian_impedance_example_controller.cpp

```diff
@@ -0,0 +1,243 @@
+#include "franka_example_controllers/cartesian_impedance_example_controller.hpp"
+#include "franka_example_controllers/pseudo_inverse.hpp"
+
+namespace franka_example_controllers
+{
+
+  CallbackReturn CartesianImpedanceExampleController::on_init()
+  {
+    // Equilibrium pose subscription
+    sub_equilibrium_pose_ = get_node()->create_subscription<geometry_msgs::msg::PoseStamped>(
+      "equilibrium_pose", 20, 
+      std::bind(&FSMImpedanceController::equilibriumPoseCallback, this, std::placeholders::_1));
```

Comment:
where is this &FSMImpedanceController coming from? Could you also provide the code for that?

Reply:
This was related to local class on our project, now equilibriumPoseCallback is memeber of &CartesianImpedanceExampleController